src/main/java/com/bobocode/nasa/service/NasaPicturesStealerService.java

```diff
@@ -0,0 +1,47 @@
+package com.bobocode.nasa.service;
+
+import com.bobocode.nasa.dto.nasa.CameraDto;
+import com.bobocode.nasa.dto.nasa.PhotoDto;
+import com.bobocode.nasa.model.Camera;
+import com.bobocode.nasa.model.Photo;
+import com.bobocode.nasa.repository.CameraRepository;
+import jakarta.transaction.Transactional;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Service
+@Transactional
```

Comment:
I used to avoid marking @Transactional the class. In my expirience it is better to mark separate methods. What about you? Do you prefer to do the same way on you work project?

Reply:
Yep. Make sense. Sometimes I prefer to use `TransactionTemplate` to have more control on transaction